tests/acceptance/features/apiSharingNg/mountOrUnmountShare.feature

```diff
@@ -86,3 +86,103 @@ Feature:  mount or unmount incoming share
       | resource      |
       | textfile0.txt |
       | folder        |
+
+
+  Scenario Outline: enable a group share sync by only one user in a group
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And group "grp1" has been created
+    And user "Alice" has been added to group "grp1"
+    And user "Brian" has been added to group "grp1"
+    And user "Alice" has disabled the auto-sync share
+    And user "Brian" has disabled the auto-sync share
+    And user "Carol" has uploaded file with content "hello world" to "/textfile0.txt"
+    And user "Carol" has created folder "FolderToShare"
+    And user "Carol" has sent the following share invitation:
+      | resource        | <resource> |
+      | space           | Personal   |
+      | sharee          | grp1       |
+      | shareType       | group      |
+      | permissionsRole | Viewer     |
+    And user "Alice" has enabled sync for share "<resource>" offered by "Carol" from "Personal" space using the Graph API
+    When user "Brian" lists the shares shared with him using the Graph API
```

Comment:
I don't see these steps relating to the scenario title.
According to the title, enable sync should be a When step

Reply:
I have changed the titile. I think it make sense now

Reply:
Also, that would make two when step so I was trying to avoid that

Reply:
hmm, but that doesn't change what When step should be

Reply:
Suggestion:
- revert scenario title
- enable sync in When step
- check the effect
- another group user checks if affected

Reply:
So, that would make two `When` step.And Also, previous code when a key of json is checked should be added again